<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>用户名</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QLineEdit" name="modify_name">
         <property name="placeholderText">
          <string>输入新用户名</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QLabel" name="disp_name">
         <property name="font">
          <font>
           <family>幼圆</family>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3" rowspan="8">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>账户操作</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="b_altername">
         <property name="text">
          <string>修改用户名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_alteremail">
         <property name="text">
          <string>修改邮箱</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_alterpassword">
         <property name="text">
          <string>修改密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_altertime">
         <property name="text">
          <string>清除播放时长</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_closedb">
         <property name="text">
          <string>关闭连接并退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>邮箱</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="disp_email">
         <property name="font">
          <font>
           <family>幼圆</family>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="modify_email">
         <property name="placeholderText">
          <string>输入新邮箱</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>密码</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="modify_passwordold">
         <property name="placeholderText">
          <string>原密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="modify_password">
         <property name="placeholderText">
          <string>新密码</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>VIP</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="disp_vip">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="disp_date">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="1" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>播放时长</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="disp_time">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="7">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>学习记录</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableView" name="table_playtime"/>
       </item>
       <item>
        <widget class="QTableView" name="table_note"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
